### ISSUE TITLE
Feature request: Sync Github issue and note

### ISSUE THREAD

**Issue by MartinBernstorff** [Frostplexx/obsidian-github-issues#20]
I'd love to have two-way sync between a github issue and a note! This'd enable me to do conceptual work in Obsidian, and then take notes as I implement in github.

Love the work you're doing here 👍

**Comment by frostplexx** [comment]
Thanks for your feedback! Could you elaborate on the two way sync a bit? Do you want to be able to create issues from within Obsidian?

**Comment by MartinBernstorff** [comment]
Absolutely! Ideally, I would have a 1:1 mapping between an Obsidian note and a GitHub issue. I.e. select repositories from a list, and then if:
* Issue added to GitHub repository "X" -> Create note in "X" folder in Obsidian Vault with data.
* Note created in "X" folder in Obsidian Vault -> Create issue with title of note in Github
* Modified on either end, sync Note <-> Issue

I recognise this is a much larger engineering task, so definitely feel no obligation! Just providing some feedback 👍
